lint/ruff.bzl

```diff
@@ -1,4 +1,4 @@
-"""API for declaring a Ruff lint aspect that visits py_library rules.
+"""API for declaring a Ruff lint aspect that visits py_library rules and filegroups tagged 'python'.
```

Comment:
I'd omit this edge case from the very short overview doc

Reply:
Done.